src/test/kotlin/com/zepben/evolve/database/paths/LocalEwbDataFilePathsTest.kt

```diff
@@ -105,25 +97,36 @@ class EwbDataFilePathsTest {
 
     @Test
     internal fun `finds specified date if it exists`() {
-        validateClosest(today, 1)
+        val descendants = mutableListOf<Path>()
+        // Files for today.
+        DatabaseType.entries.filter { it.perDate }.forEach{
+            descendants.add(Path.of(today.toString(), "${today}-${it.fileDescriptor}.sqlite"))
+        }
+
+        every { listFiles(baseDir) } answers { descendants.iterator() }
```

Comment:
todo: This pattern is repeated a few times and can probably be simplified by either:
* making the `descendants` declaration a private class member and replacing the default answer for `listFiles` with the one you have here.
* wrapping it in a function that takes the action block you are passing to `forEach` and applies it as you have here.

Reply:
Made the descendants declaration a private member